### PR TITLE
SAK-40676 - Sections enrolled by the user should be emphasized with b…

### PR DESCRIPTION
**Pull request by danielmerino**
…old font

### CODE REVIEW

**Comment by danielmerino** [comment]
@fostersdesign any objection to this patch? I'm not a CSS expert but it works pretty nice and doesn't seem to break anything else. :-)

**Comment by fostersdesign** [comment]
Your fix works, but doesn't really make semantic sense. 

The issue that you are trying to fix is that the rule needs more specificity. I'd suggest changing the rule to:
`.sectionTable .studentSectionInfo`

File: sections/sections-app/src/webapp/css/sections.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
 .selected {width:210px; text-align:right; vertical-align:top;}
 
 .transferButtonTable {text-align:center; font-weight:bold;}
-.studentSectionInfo {font-weight:bold;}
+.studentSectionInfo:last-child {font-weight:bold;}
```
Comment (fostersdesign) [inline sections/sections-app/src/webapp/css/sections.css]: This works, but I think ".sectionTable .studentSectionInfo" as the selector makes more sense semantically.

**Comment by danielmerino** [comment]
@fostersdesign I have done your required changes, could you take a look, please?

**Comment by danielmerino** [comment]
Sorry, @fostersdesign  I don't know too much about CSS. The new requested changes are done.

**Pull request merged**